### ISSUE TITLE
Missing filter value should still work

### ISSUE THREAD

**Issue by robertpainsi** [robertpainsi/filtering.js#46]
```html
<div class="filtering-group" data-group-name="available">
  <div class="filtering-filter" data-filter-name="true">Available?</div>
</div>

<div class="filtering-item" data-filter-available="true">1</div>
<div class="filtering-item" data-filter-available="">2</div>
```

- If no filter (or all filter) is checked, all items should be shown.
- If you check the filter, only the first item should be shown.

**Comment by robertpainsi** [comment]
Fixed by b783b5aec9348f90ac595884b62339c22d11db6d

Test still missing

**Comment by robertpainsi** [comment]
Now closed by https://github.com/robertpainsi/filtering.js/commit/7b5df3903958bf2a992ae2833aefd444e610c14f
